src/ontology/mondo-edit.obo

```diff
@@ -508808,7 +508812,7 @@ xref: GARD:15297 {source="OMIM:308205"}
 xref: GARD:2952 {source="Orphanet:2273"}
```

Comment:
exact synonyms with Orphanet:2273 should be reviewed

Reply:
fixed